concepts/05 Widgets/Sankey/05 Nodes and Links/05 Colorize Nodes and Links.md

```diff
@@ -39,6 +39,51 @@ When the palette does not have enough colors to paint each node differently, it
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxSankey
+            palette-extension-mode="alternate"
+            palette="Bright"
+            /* or a custom palette */
+            /* :palette="['#70c92f', '#f8ca00', '#bd1550']" */
+        >
+        </DxSankey>
+    </template>
+
+    <script>
+    import DxSankey from 'devextreme-vue/sankey';
+
+    export default {
+        components: {
+            DxSankey
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Sankey from 'devextreme-react/sankey';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <Sankey
+                    paletteExtensionMode="alternate"
+                    palette="Bright"
+                    /* or a custom palette */
+                    /* palette={['#70c92f', '#f8ca00', '#bd1550']} */
+                >
+                </Sankey>
```

Comment:
Let's make the same changes here. Make a self-closing component and write comments in JSX style:
```
<Sankey
    paletteExtensionMode="alternate"
    palette="Bright"
/>
{/* or a custom palette */}
{/* palette={['#70c92f', '#f8ca00', '#bd1550']} */}
```